docs/develop/excel-async-model.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: 'Using the Office Scripts Async APIs to support legacy scripts'
-description: 'A primer on the Office Scripts Async APIs and how to use the load/sync pattern for legacy scripts.'
-ms.date: 06/29/2020
+title: 'Support older Office Scripts that use the async APIs'
+description: 'A primer on the Office Scripts Async APIs and how to use the load/sync pattern for older scripts.'
+ms.date: 07/08/2020
 localization_priority: Normal
 ---
 
 
-# Using the Office Scripts Async APIs to support legacy scripts
+# Support older Office Scripts that use the async APIs
 
-This article will teach you how to write scripts using the legacy, async, APIs. These APIs have the same core functionality as the standard, synchronous Office Scripts APIs, but they require that your script control the data synchronization between the script and the workbook.
+This article will teach you how to maintain and update scripts that use the older, async, APIs. These APIs have the same core functionality as the standard, synchronous Office Scripts APIs, but they require that your script control the data synchronization between the script and the workbook.
```

Comment:
```suggestion
This article will teach you how to maintain and update scripts that use the older model's async APIs. These APIs have the same core functionality as the now standard, synchronous Office Scripts APIs, but they require your script to control the data synchronization between the script and the workbook.
```

Reply:
Taking this suggestion with "now-standard" instead of "now standard".